ios/test_app.rb

```diff
@@ -259,6 +259,10 @@ def make_project!(xcodeproj, project_root, target_platform, options)
     uitests_build_settings[PRODUCT_BUNDLE_IDENTIFIER] = "#{product_bundle_identifier}UITests"
   end
 
+  options[:additional_build_settings]&.each do |setting, value|
+    build_settings[setting] ||= value
```

Comment:
Is this intentional? If `build_settings[setting]` is already set, we don't set it to `value`?

Reply:
Damn, I didn't read the whole doc about `||=` operator. I think a more predictable approach would be to use `+=` and append the value (either string or array) WDYT?

Reply:
I think appending makes sense. If we had a `build_settings_overrides`, it would be safe to assume that we overwrite values.

Reply:
Changed it to `+=`